File: app/src/main/java/com/aib/net/ApiService.java
package com.aib.net;

import com.aib.entity.VideoEntity;

import io.reactivex.Observable;
import retrofit2.http.GET;

public interface ApiService {
    String videoApi = "http://api.m.mtime.cn/";


    /**
     * 视频列表
     *
     * @return
     */
    @GET("PageSubArea/TrailerList.api")
    Observable<VideoEntity> VIDEO_LIST();

}

File: app/src/main/java/com/aib/net/ApiService.kt
package com.aib.net

import com.aib.entity.BaseEntity
import io.reactivex.Observable
import retrofit2.http.Field
import retrofit2.http.FormUrlEncoded
import retrofit2.http.POST


interface ApiService {
    /**
     * 用户注册
     */
    @POST("user/register")
    @FormUrlEncoded
    fun USER_REGISTER(@Field("phone") phone: String, @Field("pwd") pwd: String): Observable<BaseEntity<String>>
}
